network_architecture_optimization/mappers/gene_mapper.py

```diff
@@ -1,19 +1,21 @@
-from abc import ABC, abstractmethod
-from typing import Callable
-
 from network_architecture_optimization.mappers.layer_description import LayerDescription
 from dataclasses import asdict
+from typing import Callable
 import tensorflow as tf
 
 
 class GeneMapper:
 
+    kernel_initializer = tf.keras.initializers.GlorotUniform(seed=None)
```

Comment:
Może lepiej nie ustawiać `seed` na `None`?

Reply:
To jest domyślna wartość i typ "initializera" w przypadku tworzenia sieci gęstych. Zostawiłem wartość None bo z taką wartością tworzona jest domyślnie.